### ISSUE TITLE
config.xml not found in the main bundle!

### ISSUE THREAD

**Issue by thomasjohnkane** [ionic-team/ionic#12724]
**Ionic version:**  (check one with "x")
[ ] **1.x** (For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[X] **3.x**

**I'm submitting a ...**  (check one with "x")
[X] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
iOS app builds but hits this error in `platforms/ios/CordovaLib/Classes/Public/CDVViewController.m`

config.xml not found in the main bundle!

on line 147

**Expected behavior:**
App finds config.xml in main bundle

**Steps to reproduce:**

- ionic start myApp (blank template)
- add ios platform
- build ios
- run app

**Related code:**

```
insert any relevant code here
```

**Other information:**

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
cli packages: (/Users/tkane/.nvm/versions/node/v6.11.0/lib/node_modules)

    @ionic/cli-utils  : 1.9.2
    ionic (Ionic CLI) : 3.9.2

global packages:

    Cordova CLI : 7.0.1

local packages:

    @ionic/app-scripts : 2.1.3
    Cordova Platforms  : ios 4.4.0
    Ionic Framework    : ionic-angular 3.6.0

System:

    ios-deploy : 1.9.1
    ios-sim    : 5.0.13
    Node       : v6.11.0
    npm        : 4.6.1
    OS         : OS X El Capitan
    Xcode      : Xcode 8.2.1 Build version 8C1002
```

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us!

I attempted to duplicate your issue following these exact steps:

```
ionic start myApp
cd myApp/
ionic cordova platform add ios
ionic cordova build ios
open platforms/ios/MyApp.xcodeproj
```

From XCode I was then able run the app on an emulator and on a device without issue. Can you provide your exact commands you used for each of your steps above?

Thank you for using Ionic

**Comment by thomasjohnkane** [comment]
Those are the exact commands. Do you see any conflicts or issues with the versions of everything I'm using listed in `ionic info`?

Here is the error I get. It appears to happen on all my projects, which makes me think it's something to do with my env versions.

```
2017-08-22 13:31:49.112572 MyApp[1823:385953] DiskCookieStorage changing policy from 2 to 0, cookie file: file:///private/var/mobile/Containers/Data/Application/0D91A6CB-17DD-478B-8344-61C1CEB0066F/Library/Cookies/Cookies.binarycookies
2017-08-22 13:31:49.190435 MyApp[1823:385953] Apache Cordova native platform version 4.4.0 is starting.
2017-08-22 13:31:49.195644 MyApp[1823:385953] Multi-tasking -> Device: YES, App: YES
2017-08-22 13:31:49.204368 MyApp[1823:385953] *** Assertion failure in -[MainViewController configFilePath], /Users/tkane/Mobile/ionic/rrn-v2/platforms/ios/CordovaLib/Classes/Public/CDVViewController.m:147
2017-08-22 13:31:49.204812 MyApp[1823:385953] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'ERROR: config.xml not found in the main bundle!'
*** First throw call stack:
(0x183cc41c0 0x1826fc55c 0x183cc4094 0x18474e82c 0x1000ae8f0 0x1000aeb34 0x1000aed74 0x1000afc04 0x1000a4c14 0x189b0d3dc 0x189b0cfa4 0x189b13750 0x189b10bf0 0x189b83414 0x1000b4750 0x1000a4930 0x189b7f42c 0x189d8fb70 0x189d958e0 0x189daa080 0x189d928c4 0x1858658bc 0x185865728 0x185865ad0 0x183c72278 0x183c71bc0 0x183c6f7c0 0x183b9e048 0x189b785dc 0x189b73360 0x1000a4820 0x182b805b8)
libc++abi.dylib: terminating with uncaught exception of type NSException
(lldb) 
```

It seems to think my settings configPath for config.xml is NULL

**Comment by kensodemann** [comment]
Here is what I am using:

```
cli packages: (/usr/local/lib/node_modules)

    @ionic/cli-utils  : 1.9.2
    ionic (Ionic CLI) : 3.9.2

global packages:

    Cordova CLI : 7.0.1 

local packages:

    @ionic/app-scripts : 2.1.3
    Cordova Platforms  : ios 4.4.0
    Ionic Framework    : ionic-angular 3.6.0

System:

    ios-deploy : 1.9.1 
    ios-sim    : 6.0.0 
    Node       : v8.3.0
    npm        : 5.3.0 
    OS         : macOS Sierra
    Xcode      : Xcode 8.3.3 Build version 8E3004b 
```

So we have basically the exact versions of everything until you get down to the system. You are on a Node version very close to the latest LTS version, so that is probably fine. XCode seems reasonably up to date as well.

You may want to try asking this question on our [forum](https://forum.ionicframework.com) or on our [slack channel](https://ionicworldwide.herokuapp.com/).

**Comment by kensodemann** [comment]
Also, could you upload the project that is failing to build to a GitHub repo where we could take a look at it?

**Comment by thomasjohnkane** [comment]
Thank you for your help, so far!

Here is the repo: https://github.com/thomasjohnkane/roofrightnow-ionic/

However, note that this is happening with the most recent starter app as well.

**Comment by kensodemann** [comment]
I cloned your repo and built it on my machine without issue. The only thing of interest I saw is that you are using older versions of the CLI and a couple of plugins. If you do an `ionic build` you will be prompted to upgrade those (I didn't for my test because I wanted to test with what you specifically had, but for my other tests I used all current versions).

I am closing this because it is not an issue with the Ionic Framework. It is most likely some sort of issue with your local setup. I suggest asking about this on our [forum](https://forum.ionicframework.com) or on our [slack channel](https://ionicworldwide.herokuapp.com/). Perhaps someone else has had the same issue and has solved it.

Thanks for using Ionic!
